appraisal.lic

```diff
@@ -25,6 +26,36 @@ class Appraisal
     @equipment_manager.items.each { |item| break unless appraise_item?(item) }
     true
   end
+  
+  def train_appraisal_with_pouches?
+    settings = get_settings
+    return false unless settings.train_appraisal_with_pouches
+    full_pouch_container = settings.full_pouch_container
+    unless full_pouch_container
+      echo "Must set full_pouch_container in YAML"
+      return false
+    end      
+    if full_pouch_container == settings.spare_gem_pouch_container
+      echo "*** Cannot have full pouches be stored where spare pouches are ***"
+      return false
+    end
+    
+    $ORDINALS.each do |ordinal|
+      
+      case bput("get #{ordinal} pouch from my #{full_pouch_container}", '^You get ', '^What were you referring')
+      when /^You get /
+        bput("appraise my pouch quick", 'Roundtime')
+        waitrt?
+        fput("put my pouch in my #{full_pouch_container}")
+      else
+        break
+      end
+      
+      break if DRSkill.getxp('Appraisal') >= 32
```

Comment:
I figured if you have the gem pouches, why not attempt to train higher. I can change this to 30.